editor/scripts/bundle.py

```diff
@@ -312,6 +312,41 @@ def get_exe_suffix(platform):
     return ".exe" if 'win32' in platform else ""
 
 
+def strip_platform_files(platform, jar):
```

Comment:
I prefer "remove", or "strip_platform_files_from_archive", since "strip" is another actual term that we use often.